test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -445,6 +447,63 @@ BOOST_AUTO_TEST_CASE(test_object_detector_train) {
 
   // Deconstructing `model` here will assert that every expected call to a
   // mocked-out method has been called.
+
+  // Test Non Maximum Suppression model
+  auto test_iou_threshold = 0.55;
+  auto test_confidence_threshold = 0.15;
```

Comment:
Nit: generally, `auto` should only be used when it improves the clarity and readability of the code, not for convenience. I think the code here would be strictly more readable with you used `double`.

Reply:
Thanks for the catch. Done!